attractions/text-field/text-field.svelte

```diff
@@ -30,8 +30,22 @@
   let inputElement;
   onMount(() => autofocus && inputElement.focus());
 
+  function toNumber(string) {
+    if (string === '') {
```

Comment:
How about `string.trim()`, because a string with just whitespaces would also return `NaN`?

Reply:
Not that it's much of a burden to add, but whitespace simply cannot make its way into an `<input type="number">`, this check was particularly to handle the case when all characters are erased from the field